apps/site/src/lib/styles/_zothacks-theme.scss

```diff
@@ -1,5 +1,10 @@
 // Common variables for site theme consistency
 
+// import bootstrap scss
+@import "bootstrap/scss/functions";
+@import "bootstrap/scss/variables";
+@import "bootstrap/scss/mixins";
+
```

Comment:
Question: Is there a reason why these imports are in this file even though they're also in `_bootstrap-utils.scss`?

Reply:
my mistake--I think I handled the merge conflicts incorrectly when I rebased.